src/components/yemen/education/juhood.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const MayaArticle = () => (
```

Comment:
Also let's stay consistent, and make all of our React component's classes and not mix some of them as functions.